seqr/utils/elasticsearch/es_search.py

```diff
@@ -824,6 +825,16 @@ def _parse_genome_versions(self, result, index_name, hit):
             'liftedOverPos': lifted_over_pos,
         })
 
+    def _parse_transloc_xstop(self, result, index_name):
+        xstop = result.pop(XSTOP_FIELD, None)
+        if xstop:
+            endChrom, end = get_chrom_pos(xstop)
```

Comment:
python variables should use underscores not snake case, so this should be end_chrom